tests/drivers/build_all/eeprom/app.overlay

```diff
@@ -51,6 +51,102 @@
 				wp-gpios = <&test_gpio 0 0>;
 				/* read-only; */
 			};
+
+			test_i2c_m24m02_a125: m24m02_a125@2 {
```

Comment:
@pdgendt @erwango Did you mean like this, and leaving the additions to eeprom_at2x.c out?